services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -1,62 +1,80 @@
+/* eslint no-debugger: 0 */
+
 import React from 'react';
 import Head from 'next/head';
+import PropTypes from 'prop-types';
+import Applications from './Applications';
 
 import { SectionHeader, PUBLIC_CSS_URL } from '@cityofboston/react-fleet';
-
 import fetchCommissions, {
   Commission,
 } from '../../client/graphql/fetch-commissions';
-
 export interface Props {
   commissions: Commission[];
 }
-
 export default class IndexPage extends React.Component<Props> {
   static async getInitialProps(): Promise<Props> {
     const commissions = await fetchCommissions();
     return { commissions };
   }
+  constructor(props) {
+    super();
+    this.state = {
+      firstEmail,
+      secondEmail,
+      emailError: '',
+      emailMatch: false,
+      dirty: false,
+    }; /*use this for submit */
 
-  fullName: HTMLInputElement | null = null;
+    this.handleBlur = this.handleBlur.bind(this);
```

Comment:
Rather than using `bind` in the constructor, you can write this in the class as:

```
handleBlur = (ev) => {
  …
}
```

(applies to the other `bind`s below)

Reply:
This should let you delete the constructor entirely.